src/forms/redux-form/TextInput.jsx

```diff
@@ -8,9 +8,10 @@ import TextInput from '../TextInput';
  * @return {Component} TextInput
  */
 const ReduxFormTextInput = props => {
-	const { meta, input, ...other } = props;
+	const { meta, input, validateAfterTouched, ...other } = props;
+	const error = (!validateAfterTouched || meta.touched) ? meta.error : null;
```

Comment:
Should this be `validateBeforeTouched` instead? Just thinking what the most common use case might be so we don't have to add an extra prop every time. I would think we would in most cases, want always to check `touched` before validating.

Reply:
also if we can add some documentation somewhere as to what this property is there for, that would be 💯

Reply:
> Should this be validateBeforeTouched instead?

Actually I think that makes more sense too 👍

Reply:
I'll make this update to both my prs!